src/components/Search.jsx

```diff
@@ -155,6 +155,9 @@ export const Search = ({
                         ref: inputReference,
                         onFocus: handleInputFocus,
                         onBlur: () => setShowShortcutKey(true),
+                        onChange: (e) => {
+                            setInputValue(e.target.value);
+                        },
```

Comment:
onChange : (e) => setInputValue(e.target.value)

what does it do, if there is no value in the input ?